src/Nethermind/Nethermind.Blockchain/Receipts/KeccaksIterator.cs

```diff
@@ -20,17 +30,17 @@ public KeccaksIterator(Span<byte> data)
             Index = -1;
         }
 
-        public bool TryGetNext(out KeccakStructRef current)
+        public bool TryGetNext(out KeccakIteratorRef current)
```

Comment:
Hmm... why? Is it because of removal of `KeccakStructRef`?

Reply:
`readonly ref` as a parameter is `in`, can't pass it `out` :(

So need so pass in another structure that can pass `out`